lib/lita/handlers/jarvis.rb

```diff
@@ -0,0 +1,21 @@
+require "jarvis/commands/merge"
+require "jarvis/commands/bounce"
+require "jarvis/mixins/fancy_route"
+
+module Lita
+  module Handlers
+    class Jarvis < Handler
+      extend ::Jarvis::Mixins::FancyRoute
+
+      on :loaded, :loaded
+      def loaded(*args)
+        WorkPool.setup_singleton
+      end
```

Comment:
If you use the singleton module you won't need to do this.
It will be done of the first call. Unless you think we might have a concurrency issue when accessing the singleton for the first time?